### PR TITLE
Feature/ws integration 2

### PR DESCRIPTION
**Pull request by anistal**
## Description
The Protocol used to manage Khermes is full implemented. You can open http://localhost:8080 and write your commands in the textarea to test.

This is an example of commands that you can write:

```
[command]
create twirl-template
[name]
template1
[content]
@import com.stratio.khermes.helpers.faker.Faker

@(faker: Faker)
{
  "name" : "@(faker.Name.firstName)"
}
############################################################
[command]
create kafka-config
[name]
kafka1
[content]
kafka {
  bootstrap.servers = "localhost:9092"
  acks = "-1"
  key.serializer = "org.apache.kafka.common.serialization.StringSerializer"
  value.serializer = "org.apache.kafka.common.serialization.StringSerializer"
}
############################################################
[command]
create generator-config
[name]
generator1
[content]
khermes {
  templates-path = "/tmp/hermes/templates"
  topic = "alicia"
  template-name = "chustasTemplate"
  i18n = "ES"

  timeout-rules {
    number-of-events: 10
    duration: 2 seconds
  }

  #stop-rules {
  #  number-of-events: 5000
  #}
}
############################################################
[command]
start
[twirl-template]
template1
[kafka-config]
kafka1
[generator-config]
generator1
############################################################
[command]
ls
############################################################
[command]
stop
[node-ids]
51a50bf6-a919-4c04-830a-b3fc7e1112bf
```

### Testing
- [ ] Unit, integration, acceptance tests as appropriate
- [ ] Coverage (>90%)

### Documentation
- [ ] Changelog update
- [ ] Documentation entry

### Scalastyle
Result of scalastyle execution:

### CODE REVIEW

File: src/test/scala/com/stratio/khermes/clients/http/protocols/WSProtocolMessageTest.scala [hunk 1]
```diff
@@ -20,23 +20,25 @@ import org.junit.runner.RunWith
...
+        |multiline line 2 content
       """.stripMargin
 
-    WsProtocolCommand.parseTextBlock(block)
+    val result = WsProtocolCommand.parseTextBlock(block)
+    result should be(WSProtocolMessage(WsProtocolCommand.Ls, Map(
+      "arg1" -> "one line content",
+      "arg2" -> "multiline line 1 content\nmultiline line 2 content"
+    )))
   }
```
Comment (albertostratio) [inline src/test/scala/com/stratio/khermes/clients/http/protocols/WSProtocolMessageTest.scala:43]: Miss a lot of tests in this class, I guess coverage has decreasead a lot :( It's ok for now but we should add more tests in the future.

File: src/main/scala/com/stratio/khermes/clients/shell/KhermesConsoleHelper.scala [hunk 2]
```diff
@@ -29,10 +29,10 @@ case class KhermesConsoleHelper(client: KhermesClientActor) {
   lazy val reader = createDefaultReader()
 
   parseLines(
-    firstLoad(AppConstants.KhermesConfigNodePath),
-    firstLoad(AppConstants.KafkaConfigNodePath),
-    firstLoad(AppConstants.TemplateNodePath),
-    firstLoad(AppConstants.AvroConfigNodePath)
+    firstLoad(AppConstants.GeneratorConfigPath),
```
Comment (albertostratio) [inline src/main/scala/com/stratio/khermes/clients/shell/KhermesConsoleHelper.scala:32]: What does this method do? Would you mind changing the name of the method to be self-explanatory?

**Review by albertostratio — CHANGES_REQUESTED** [review]
Great job @anistal !!

File: src/main/scala/com/stratio/khermes/clients/shell/KhermesConsoleHelper.scala (hunk 2 above)
Comment (anistal) [reply src/main/scala/com/stratio/khermes/clients/shell/KhermesConsoleHelper.scala:32]: this method "firstLoad" was created by @eruizgar if my memory don't fail. I think that it creates paths if there are not created the first time. Check with him for more information.

File: src/test/scala/com/stratio/khermes/clients/http/protocols/WSProtocolMessageTest.scala (hunk 1 above)
Comment (anistal) [reply src/test/scala/com/stratio/khermes/clients/http/protocols/WSProtocolMessageTest.scala:43]: Yes. I was developing with few tests to have something that works for Miami :dancer: In general the application needs more tests. Anyway the coverage of the class over test is almost 100 percent.

**Review by eambrosio — APPROVED** [review]
Awesome @anistal!!!!
LGTM :+1:

**Review by albertostratio — APPROVED** [review]

**Pull request merged**